test/proposal/Proposal.t.sol

```diff
@@ -231,4 +359,23 @@ contract ProposalTest is Test {
         types.assumeElemNotInSet(modules, address(paymentProcessor));
         types.assumeElemNotInSet(modules, address(token));
     }
+
+    function cutArray(uint size, address[] memory addrs)
+        internal
+        pure
+        returns (address[] memory)
+    {
+        uint length = addrs.length;
+        vm.assume(length > 0); //Array has to be at least 1
+
+        if (length <= size) {
+            return addrs;
+        }
+
+        address[] memory cutArry = new address[](size);
+        for (uint i = 0; i < size - 1; i++) {
```

Comment:
unchecked pre-increments can make the tests run faster.

Also no need to initialize the `uint i` variable.